apps/desktop-wallet/src/modals/ModalContainer.tsx

```diff
@@ -79,7 +79,7 @@ export default styled(ModalContainer)<{ hasPadding?: boolean }>`
   left: 0;
   display: flex;
   padding: ${({ hasPadding }) => hasPadding && 'var(--spacing-4)'};
-  z-index: 1;
+  z-index: 2;
```

Comment:
Possibly this is not needed after the merge of the previous PR?

Reply:
yep! Removing the z-index prop ;)